simpleclient_dropwizard/src/main/java/io/prometheus/client/dropwizard/DropwizardExports.java

```diff
@@ -0,0 +1,202 @@
+    }
+
+    /**
+     * Convert a timer to an histogram and use the TimeUnit specified in the constructore to
+     * compute the appropriate factor to apply to histogram values.
+     */
+    List<MetricFamilySamples> fromTimer(String name, Timer timer) {
+        return fromSnapshotAndCount(name, timer.getSnapshot(), timer.getCount(),
+                1.0D / timeUnit.toNanos(1L));
+    }
+
+    /**
+     * Export a Meter to prometheus Counters to multiple prometheus metrics:
+     * - a Gauge to total meter count
+     * - Counters for the following metric values:
+     *   - 1 minute rate
+     *   - 5 minute rate
+     *   - 15 minutes rate
+     *   - mean rate
+     */
+    List<MetricFamilySamples> fromMeter(String name, Meter meter) {
+        return Arrays.asList(
+                new MetricFamilySamples(name + "_total", Type.COUNTER, GENERIC_HELP_MESSAGE,
+                        Arrays.asList(new MetricFamilySamples.Sample(name + "_total",
+                                new ArrayList<String>(),
+                                new ArrayList<String>(),
+                                meter.getCount()))),
+                new MetricFamilySamples(name + "_1m_rate", Type.GAUGE, GENERIC_HELP_MESSAGE,
+                        Arrays.asList(new MetricFamilySamples.Sample(name + "_1m_rate",
+                                new ArrayList<String>(),
+                                new ArrayList<String>(),
+                                meter.getOneMinuteRate()))),
+                new MetricFamilySamples(name + "_5m_rate", Type.GAUGE, GENERIC_HELP_MESSAGE,
+                        Arrays.asList(new MetricFamilySamples.Sample(name + "_5m_rate",
+                                new ArrayList<String>(),
+                                new ArrayList<String>(),
+                                meter.getFiveMinuteRate()))),
+                new MetricFamilySamples(name + "_15m_rate", Type.GAUGE, GENERIC_HELP_MESSAGE,
+                        Arrays.asList(new MetricFamilySamples.Sample(name + "_15m_rate",
+                                new ArrayList<String>(),
+                                new ArrayList<String>(),
+                                meter.getFifteenMinuteRate()))),
+                new MetricFamilySamples(name + "_mean_rate", Type.GAUGE, GENERIC_HELP_MESSAGE,
+                        Arrays.asList(new MetricFamilySamples.Sample(name + "_mean_rate",
+                                new ArrayList<String>(),
+                                new ArrayList<String>(),
+                                meter.getMeanRate())))
+
+        );
+    }
+
+    @Override
+    public List<MetricFamilySamples> collect() {
+        ArrayList<MetricFamilySamples> mfSamples = new ArrayList<MetricFamilySamples>();
+
+        for (SortedMap.Entry<String, Gauge> entry : registry.getGauges().entrySet()) {
+            mfSamples.addAll(fromGauge(entry.getKey(), entry.getValue()));
```

Comment:
The names will tend to contain periods, you need to sanitize these are they aren't valid metric names in Prometheus.